### ISSUE TITLE
Create metric for estimated number of hours to finish solving packages per solver

### ISSUE THREAD

**Issue by pacospace** [thoth-station/slo-reporter#275]
**Is your feature request related to a problem? Please describe.**
As Thoth User and Maintainer,

I want to know how much time thoth will take to solve the new solver added to the system.

**Describe the solution you'd like**
Create metric for estimated number of hours to finish solving packages per solver:

- [ ] Introduce estimated_time_to_solve [h] = unsolved_packages [packages] / learning_rate [packages/h] over 7 days period.

**Describe alternatives you've considered**

**Additional context**

**Comment by pacospace** [comment]
This is part  of Grafana already, do we want it also in the weekly report?

**Comment by goern** [comment]
no I dont think so, the weekly SLO mail should be user-facing SLO, this metric seems to be an operational metric that is for our own observations.

syn?

**Comment by pacospace** [comment]
Sure, let's close this issue!
